src/components/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,199 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import { datePickerStyles } from './DatePicker.style';
+import useTheme from 'hooks/useTheme';
+import DayPickerInput from 'react-day-picker/DayPickerInput';
+import { DateUtils, RangeModifier, DayPickerInputProps } from 'react-day-picker';
+import 'react-day-picker/lib/style.css';
+import dayjs from 'dayjs';
+
+import TextField from '../TextField';
+import Icon from '../Icon';
+import { wrapperStyle } from '../TextField/TextField.style';
+import { flex } from '../../theme/functions';
```

Comment:
Import can be shortened to `theme/functions`

Reply:
i don't care if its going to be relative or absolute path to be honest as both serve the same purpose and it will work at the end :)  but we can decide one way or another